types/types.pb.go

```diff
@@ -74,6 +76,10 @@ func (m *Coin) GetDenom() string {
 	return ""
 }
 
+func (*Coin) XXX_MessageName() string {
```

Comment:
Why are we introducing `XXX_*`?

Reply:
Sorry, I had a different cosmos proto binary on my system. Just regenerated the proto files and pushed. This should be in the grpc PR instead because they are used by grpc for reflection.